wiki/People/The_Team/en.md

```diff
@@ -61,25 +61,25 @@ The following are user groups which are made up of the osu! community members th
 | :-- | :-- |
 | ![][flag_CA] [awp](https://osu.ppy.sh/users/2650) | General housekeeper and helper, chibi artist. [Barrel Roll Weekly](http://brw.twinkfish.com/) |
 | ![][flag_NO] [cYsmix](https://osu.ppy.sh/users/272870) | Audio designer, [official osu! Featured Artist](https://osu.ppy.sh/beatmaps/artists/2) |
-| ![][flag_NZ] [deadbeat](https://osu.ppy.sh/users/128370) | World Cup organizer and administrator |
+| ![][flag_NZ] [deadbeat](https://osu.ppy.sh/users/128370) | World Cup organiser and administrator |
 | ![][flag_US] [Derekku](https://osu.ppy.sh/users/91341) | General housekeeper, community manager |
 | ![][flag_NZ] [Echo](https://osu.ppy.sh/users/431) | osu! developer, provider of IRC integration for in-game chat, site maintainer. [Blog](http://blog.echo.sh/) |
 | ![][flag_US] [HappyStick](https://osu.ppy.sh/users/256802) | osu! Coffee Hour host, World Cup streamer, tournament organizer |
 | ![][flag_NL] [Intermezzo](https://osu.ppy.sh/users/136842) | osu! developer, provider of osz2 and p2p backend |
 | ![][flag_US] Jim | Original site designer, hosting provider in the early days. [Brave New Games](http://www.bravegamer.com/) |
-| ![][flag_AR] [juankristal](https://osu.ppy.sh/users/443656) | World Cup organizer and administrator |
-| ![][flag_DE] [Loctav](https://osu.ppy.sh/users/71366) | World Cup organizer and administrator, QAT leader, community manager |
+| ![][flag_AR] [juankristal](https://osu.ppy.sh/users/443656) | World Cup organiser and administrator |
```

Comment:
WalterToro/deadbeat/juankristal/Loctav are no longer actively involved in WC organization, maybe change it to  `Former World Cup organiser and administrator` or something similar?

Reply:
this is part of the "Inactive" section

Reply:
agree with clayton